### PR TITLE
docs(App Structure): Simplify readme.md

### PR DESCRIPTION
**Pull request by joshuef**
QA:

- Do links to web app doc work from `README.md`
- Do links to application design doc work from `README.md`

### CODE REVIEW

File: docs/web-app-development/SAFE-Web-App-Overview.md [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+
+![Experimental features/APIs toggle](resources/readme/experiments-toggle.png)
```
Comment (bochaco) [inline docs/web-app-development/SAFE-Web-App-Overview.md]: The image path is incorrect

File: docs/web-app-development/SAFE-Web-App-Overview.md [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+
+When the experimental APIs/features are enabled with this toggle, a visual indicator is displayed on the right hand side of the URL address bar. If you don't see this indicator it means the experimental APIs are not enabled or available to neither the browser nor to any website/webapp.
+
+![Experimental features/APIs toggle](resources/readme/experiments-visual-indicator.png)
```
Comment (bochaco) [inline docs/web-app-development/SAFE-Web-App-Overview.md]: The image path is incorrect

File: docs/web-app-development/SAFE-Web-App-Overview.md [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+
+### Support for XOR-URLs:
+
+Not only does the DOM API allow SAFE applications to link/reference any MutableData/ImmutableData on the network using its XOR-URL (URL based on its XoR name location on the network), but it also enables the SAFE Browser to fetch any website, webapp, or random file from the network using a XOR-URL. Note this is in addition to being able to fetch content using the SAFE DNS (Decentralised Naming System) with publicName-URLs, e.g. `safe://subname.pubname`.
+
+Even HTML tags can reference content from a src attribute using XOR-URLs, e.g. an image tag can have a XOR-URL in the src attribute, and the browser is capable of fetching it and render it as usual.
+
+As shown in the screenshot below, the SAFE Browser is capable of decoding the XOR-URL to extract the location of the content, content type (if the mimeType is also encoded in the XOR-URL), fetch it, and render it accordingly as it does it with any publicName-URL (i.e. the URLs which make use of the SAFE DNS system):
+
+![XOR-URL of HTML file](resources/readme/xorurl-screenshot.png)
```
Comment (bochaco) [inline docs/web-app-development/SAFE-Web-App-Overview.md]: The image path is incorrect

File: docs/web-app-development/SAFE-Web-App-Overview.md [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+
+A good resource to learn and understand a bit more about XOR-URLs (as a user) is by visiting this little tutorial where it explains different cases for how to share websites and files with publicName-URLs and XOR-URLs, just access the following safe-URL and follow the links: [safe://xorurl.tutorials](safe://xorurl.tutorials)
+
+Please note that since the XOR-URLs are still in its experimental stage you'll need to switch the toggle form the browser to be able to use them to fetch content.
+
+For more details/information about the XOR-URLs, you can read the proposed [RFC](https://github.com/maidsafe/rfcs/blob/357384147ae005e4061079b27a30f43cf379fda5/text/0000-xor-urls/0000-xor-urls.md), or the [discussions taking place in our DevForum](https://forum.safedev.org/t/xor-address-urls-xor-urls/1952), or by watching this [screencast](https://www.youtube.com/watch?v=BikfxRNARnM).
+
```
Comment (bochaco) [inline docs/web-app-development/SAFE-Web-App-Overview.md]: We can keep this video and/or we can link it to the new one we recorded with the community

File: docs/web-app-development/SAFE-Web-App-Overview.md [hunk 5]
```diff
@@ -0,0 +1,152 @@
...
+- [Patter (safe://patter.dapp)](safe://patter.dapp), which is a proof-of-concept decentralised Twitter clone ([source code](https://github.com/maidsafe/safe-patter-js))
+- The [WebID Mgr. proof-of-concept webapp (safe://webidmgr.dapp)](safe://webidmgr.dapp)([source code](https://github.com/maidsafe/safe-web-id-manager-js))
+
+### The MutableData viewer
+
+When a URL (either publicName-URL or XOR-URL) is targeting a `MutableData`, which happens to not have an "index.html" file, or which is probably not even a files container (a.k.a. an NFS container) but simply a `MutableData` created by any application to store data on the network, a `MutableData` viewer is rendered by the SAFE Browser. This web-based viewer displays the list of `MutableData` entries, showing the key, version and value of each of them, and it also displays the permissions sets defined for such a `MutableData`. This is a very basic first version, which can hopefully evolve into a SAFE data browser in the future.
+
+![XOR-URL of HTML file](resources/readme/md-viewer.png)
```
Comment (bochaco) [inline docs/web-app-development/SAFE-Web-App-Overview.md]: The image path is incorrect

File: docs/web-app-development/SAFE-Web-App-Overview.md [hunk 6]
```diff
@@ -0,0 +1,152 @@
...
+
+eg, on OSX:
+
+```bash
+open SAFE Browser.app --args --live
+```
+
+When the SAFE Browser launched is the one for the mock network, a visual indicator is displayed in the address bar, on the left hand side of the URL address bar as shown in the following image:
+
+![Mock Network visual indicator](resources/readme/mock-visual-indicator.png)
```
Comment (bochaco) [inline docs/web-app-development/SAFE-Web-App-Overview.md]: The image path is incorrect

**Comment by joshuef** [comment]
Updated with fixed image links.

File: README.md [hunk 7]
```diff
@@ -215,33 +68,15 @@ There are a few build commands for various situations:
...
-
-`webpack.config.base` contains loaders and alias' used across all webpack configs.
-
-There is a prod, config. Alongside renderer configs.
-
-When developing against hot reloading, the `vendor` setup is used to speed up build times etc.
-
-There are 'dev' mode configs for running against the NODE_ENV=develeopment setup.
-There are 'live-dev' configs for running against NODE_ENV=production but without needing to package.
+for simple data flow and clear componentisation.
```
Comment (hunterlester) [inline README.md]: Straggling slice

**Comment by hunterlester** [comment]
Love this documentation. Really nice, Josh.

**Comment by joshuef** [comment]
Updated with that slice removed and video link pointing to new browser overview screencast

**Comment by safesurfer** [comment]
@joshuef  WebApp doc from `README.md` works fine. but Browser Application Design doc gives `404 page not found` error

**Comment by safesurfer** [comment]
QA verified

**Pull request merged**